crates/driver/src/util/serialize.rs

```diff
@@ -0,0 +1,30 @@
+use serde::{Deserialize, Serialize};
+
+/// Serializes [`ethereum_types::U256`] as a decimal string.
```

Comment:
I saw these change to `serde_as` in the other PR (which I think is more correct).